src/query/storage/converter.go

```diff
@@ -278,28 +278,32 @@ func iteratorToTsSeries(
 	iter encoding.SeriesIterator,
 	enforcer cost.ChainedEnforcer,
 	tagOptions models.TagOptions,
-) (*ts.Series, error) {
+) (*ts.Series, *ts.Exemplars, error) {
```

Comment:
Should this be `ts.Exemplars` since it's a slice?
nit: make a `traceSeries` struct containing `*ts.Series` and `ts.Exemplars`?